documents/papernach.md

```diff
@@ -40,7 +41,25 @@ $api->order->create(array('amount' => 0,'currency' => 'INR','method' => 'nach','
 
 **Parameters:**
 
-All parameters listed [here](https://razorpay.com/docs/api/route/#create-transfers-from-payments) are supported
```

Comment:
https://razorpay.com/docs/api/payments/recurring-payments/paper-nach/create-authorization-transaction/#112-create-an-order

Reply:
its there but its not showing check this
https://github.com/razorpay/razorpay-php/blob/papernach_doc_correction/documents/papernach.md?plain=1#L51